common/src/main/java/com/wynntils/wynn/item/parsers/WynnItemMatchers.java

```diff
@@ -165,11 +165,15 @@ public static boolean isCraftedGear(ItemStack itemStack) {
 
     public static boolean isMythic(ItemStack itemStack) {
         // only gear, identified or not, could be a mythic
-        if (!(isUnidentified(itemStack) || isGear(itemStack))) return false;
+        if (!(isUnidentified(itemStack) || isGear(itemStack)) || !isMythicBox(itemStack)) return false;
```

Comment:
Also why did this get changed this?

Reply:
I agree, this is pretty tough to understand. And it does not seem to be used? 

Why was this method not used instead? I don't think it is likely that people go throwing out their mythical gear on the ground, but in the odd case that they actually do, I think it could just as well be scaled up, so not only boxes (unidentified mythics) but also identified mythics could be scaled. 

On the third hand, the isMythicBox implementation seems much more efficient, and I presume this will be called often when rendering items on the ground. So maybe we should go the other way around, making isMythic something like this:

```
if (isMythicBox(itemStack) return true;
if (!isGear(itemStack)) return false;

return itemStack.getHoverName().getString().contains(ChatFormatting.DARK_PURPLE.toString());
```

Reply:
The `isMythicBox` is a check that it has the texture of a unid mythic box by checking it is stone shovel with damage value 6. The `isMythic` method does not work for items on the ground due to differences from wynncraft.

> but also identified mythics could be scaled.
This is simply different from mythic boxes since it has a different texture, and would require a redicussing with the moderators I think